app/src/main/res/values/strings.xml

```diff
@@ -696,7 +696,7 @@
     <string name="clear_open_file_summary">Clears selected default file opening apps</string>
     <string name="ftp_readonly">Read-only access</string>
     <string name="legacy_listing_title">Use legacy listing for root</string>
-    <string name="legacy_listing_summary">If enabled, uses stat for listing, else uses ls</string>
+    <string name="legacy_listing_summary">If enabled, uses old method (ls) for listing files, else uses stat (new method)</string>
```

Comment:
Old method and new method is kinda redundant. Plus the ls and stat is something user wouldn't care. We should minimise text here.
Say 'If enabled uses legacy listing method, else uses new method'

Reply:
I'll then say **If enabled, uses legacy method to list files**.